.toolversions

```diff
@@ -0,0 +1,22 @@
+cmake="
+  DeclaredVersion='3.6.0'
+  DownloadFileLinux='cmake-3.6.0-Linux-x86_64.tar.gz'
+  DownloadFileOSX='cmake-3.6.0-Darwin-x86_64.tar.gz'
+  DownloadFileWindows32='cmake-3.6.0-win32-x86.zip'
+  DownloadFileWindows64='cmake-3.6.0-win64-x64.zip'
+  DownloadUrl='https://cmake.org/files/v3.6/'
+  LocalToolFolder='Tools/downloads/cmake'
+  LocalToolFolderWindows='Tools\\downloads\\cmake'
+  LocalSearchPathLinux='cmake-3.6.0-Linux-x86_64/bin/cmake'
+  LocalSearchPathOSX='cmake-3.6.0-Darwin-x86_64/CMake.app/Contents/bin/cmake'
+  LocalSearchPathWindows32='cmake-3.6.0-win32-x86\\bin\cmake.exe'
+  LocalSearchPathWindows64='cmake-3.6.0-win64-x64\\bin\cmake.exe'
+  ProgramFilesToolPath32='%programfiles(x86)%\\CMake\\bin\\cmake.exe'
+  ProgramFilesToolPath64='%programfiles%\\CMake\\bin\\cmake.exe'
+  VSInstallToolPath32='%programfiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\IDE\\CommonExtensions\\Microsoft\\CMake\\CMake\\bin\\cmake.exe'
+  VSInstallToolPath64='%programfiles(x86)%\\Microsoft Visual Studio\\2017\\Professional\\Common7\\IDE\\CommonExtensions\\Microsoft\\CMake\\CMake\\bin\\cmake.exe'
```

Comment:
Sorry my mistake. These names should be `VSProfessionalToolPath` and 'VSEnterpriseToolPath' or similar. What editions of VS include CMake is yet to be found out (by me).